middleware.js

```diff
@@ -118,12 +127,12 @@ module.exports = function(options){
 
           // If response is falsey, also write to file
           if (!options.response) {
-              mkdirp(dirname(cssPath), 0700, function(err){
-                  if (err) return error(err);
-                  fs.writeFile(cssPath, data, 'utf8', function(err) {
-                      if (err) return error(err);
-                  });
+            mkdirp(dirname(cssPath), '0700', function(err){
+              if (err) return error(err);
+              fs.writeFile(cssPath, data, 'utf8', function(err) {
+                if (err) return error(err);
```

Comment:
Lets try to avoid K&R style in favour of proper bracketing:

```javascript
if (err) {
  return error(err);
}

fs.writeFile(cssPath, data, 'utf8', function(err) {
  if (err) {
    return error(err);
  }
```

The extra line between first if-block and `fs.writeFile ..` is not the result of OCD, it is just a preferred design! :wink: